src/summary/patient-chart-summary.css

```diff
@@ -7,3 +7,18 @@
   justify-content: center;
   padding-top: 3.25rem;
 }
+
+.patientChartCards {
+  padding: 3.25rem 0;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  width: 800px;
+  justify-content: flex-start;
+}
+
+.h {
```

Comment:
A more descriptive name for the `.h` class

Reply:
@jonathandick ?

Reply:
renamed to patientChartCardsContainer